docs/source/ref/models/index.rst

```diff
@@ -11,3 +11,4 @@ Model API reference.
    querysets
    models
    indexes
+   search
```

Comment:
need a mention on docs/source/index.rst too (have forgotten this on some recent changes)